test/test_convert.py

```diff
@@ -78,16 +122,54 @@ def test_igem_ludox(self):
         assert filecmp.cmp(markdown_output, markdown_comparison_file), "Markdown files are not identical"
         print('File identical with test file')
 
-        # Check Autoprotocol output
-        autoprotocol_comparison_file = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                                'testfiles', 'igem_ludox_autoprotocol.json')
+    def test_igem_ludox_opentrons(self):
```

Comment:
This is the entry point to running the opentrons specialization.  It can be run via pytest.